python/ray/serve/_private/deployment_state.py

```diff
@@ -1581,18 +1581,6 @@ def deploy(self, deployment_info: DeploymentInfo) -> bool:
                     deployment_info.target_capacity_direction,
                 )
 
-            downscale_smoothing_factor = (
```

Comment:
Is this accidentally removed? I know it's just suggesting a better config, but is it no longer applies so we decide to take it out?

Reply:
Oh yeah I added this warning back when a small smoothing factor could (basically) permanently block downscaling. With the smoothing factor fix [here](https://github.com/ray-project/ray/pull/42394) we shouldn't need this warning anymore.